src/Tempest/Console/src/Components/Renderers/ChoiceRenderer.php

```diff
@@ -17,7 +19,9 @@ final class ChoiceRenderer
     public function __construct(
         private bool $multiple = true,
         private int $maximumOptions = 10,
-        private ?string $default = null,
+        // TODO: support defaults
+        /** @phpstan-ignore-next-line */
+        private null|Stringable|UnitEnum|string $default = null,
```

Comment:
PHPStan is complaining about this default not being used. Indeed it isn't. Is that because it's still WIP, or should default handling be moved somewhere else?

Reply:
That was an overlook, which I just fixed, thanks!